### PR TITLE
fix(GRA-919): use netmaker logo

### PR DESCRIPTION
**Pull request by Aceix**
TESTING STEPS
1. build netclient gui
2. run the gui
3. check that the netmaker logo shows as the app icon (may not work on GNOME as an extra .desktop file may be needed)

### CODE REVIEW

File: gui/utils.go [hunk 1]
```diff
@@ -23,3 +24,13 @@ func OpenUrlInBrowser(url string) error {
 
 	return err
 }
+
+// GetFileAsBytes returns the byte array form files
+// It can be used return th app icon
```
Comment (0xdcarns) [inline gui/utils.go:29]: return the*

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Comment by theguy951357** [comment]
will wait on changes

File: gui/utils.go (hunk 1 above)
Comment (Aceix) [reply gui/utils.go]: resolved

**Review by mattkasun — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
windows GUI shows netmaker logo
Mac GUI has netmaker logo
pass

**Pull request merged**